packages/build/package.json

```diff
@@ -66,7 +66,7 @@
     "@bugsnag/js": "^7.0.0",
     "@netlify/cache-utils": "^5.0.1",
     "@netlify/config": "^20.0.0",
-    "@netlify/edge-bundler": "4.0.2",
+    "@netlify/edge-bundler": "^4.1.0",
```

Comment:
Can we leave it pinned so that future updates don't break stuff again?

Reply:
Done in https://github.com/netlify/build/pull/4701/commits/4d9605f3316be926ef0841f48624419c02ead0da.